src/operations/write.ts

```diff
@@ -69,11 +69,20 @@ export const write = (
 
   if (operationName === 'Query') {
     writeSelection(ctx, operationName, select, data);
+  } else if (operationName === 'Subscription') {
```

Comment:
Is there any reason for this not to use `writeRoot`?

Reply:
Well we don't add typenames to subscriptions so I didn't think there was much use to using the normal way

Reply:
We do 😅  just not to the toplevel. We don't need the toplevel typename for any operations now anyway though.

Reply:
When the subscription for a new message triggers __typename is not included on the message either

Reply:
just checked and the `__typename` field is correctly added to the query (in urql v1.4, but should be added in <1.4 as well) so what are we receiving in responses?